okhttp/src/main/kotlin/okhttp3/internal/connection/ForceConnectRoutePlanner.kt

```diff
@@ -0,0 +1,22 @@
+package okhttp3.internal.connection
+
+import okhttp3.Address
+import okhttp3.HttpUrl
+
+/**
+ * A RoutePlanner that will always generate a ConnectPlan, ignoring any connection pooling
+ */
+class ForceConnectRoutePlanner(private val delegate: RealRoutePlanner) : RoutePlanner {
```

Comment:
I don't think we should allow this on Android by default. Network traffic while the app is backgrounded (no activity, service, or workmanager) should be avoided.

Or if network is lost.

I guess either
- document and failfast on android
- add some predicate that can let it work on android, such as isForegrounded.

Reply:
Got it, I'll figure something out